File: micro-bit/js/1-3.js
let count = 0;
basic.showIcon(IconNames.Heart);

input.onButtonPressed(Button.A, function button_a() {
    count++;

    if (count > 2) {
        count = 0;
    }

    if (count == 0) {
        basic.showIcon(IconNames.Heart);
    } else if (count == 1) {
        basic.showIcon(IconNames.No);
    } else if (count == 2) {
        basic.showIcon(IconNames.Square);
    }

})

File: micro-bit/python/1-3.py
count = 0
basic.show_icon(IconNames.HEART)

def button_a():
    global count
    count+=1

    if count > 2:
        count = 0

    if count == 0:
        basic.show_icon(IconNames.HEART)
    elif count == 1:
        basic.show_icon(IconNames.NO)
    elif count == 2:
        basic.show_icon(IconNames.SQUARE)

input.on_button_pressed(Button.A, button_a)